vpr/src/base/clustered_netlist.h

```diff
@@ -169,12 +175,19 @@ class ClusteredNetlist : public Netlist<ClusterBlockId, ClusterPortId, ClusterPi
         bool net_is_global(const ClusterNetId id) const;
 
     public: //Public Mutators
+
         //Create or return an existing block in the netlist
         //  name        : The unique name of the block
         //  pb          : The physical representation of the block
         //  t_type_ptr  : The type of the CLB
         ClusterBlockId create_block(const char *name, t_pb* pb, t_type_ptr type);
 
+        //Add the equivalent block type for a CLB
+        //  blk_id      : The block placed in an equivalent tile location
+        //  i_eq_type   : The index used to retrieve the equivalent tile from the t_type_ptr structure
```

Comment:
Why both the index and the type?

Reply:
Basically because when setting an equivalent type for a CLB I must update both the `vector_map`s relative to the Block (index and type `vector_maps`. I found it cleaner to have separate `vector_map`s to hold indices and actual types.